pages/sync/github.en.mdx

```diff
@@ -81,6 +81,11 @@ If you use multiple token sets you might also want to create different token fil
 
 Don't want to include the global set in the output but use it for resolving aliases? Provide another parameter like this and the transformer will exclude that in the output: `node token-transformer tokens.json output.json global,dark global`
 
+### 8. Using multi-file sync
```

Comment:
One small thing here: This should probably be a subheading to section 7, as it's still about token-transformer